packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -140,51 +156,64 @@
 
 :host .heading,
 :host .description {
-  @apply flex w-full;
+  @apply flex;
+  inline-size: var(--calcite-container-size-content-fluid);
 }
 
 :host .heading {
-  @apply text-color-2 font-medium;
+  @apply font-medium;
+  color: var(
+    --calcite-accordion-item-heading-text-color,
+    var(--calcite-accordion-item-text-color, var(--calcite-color-text-2))
+  );
 }
 :host .description {
-  @apply text-color-3 mt-1;
+  @apply mt-1;
+  color: var(
+    --calcite-accordion-item-description-text-color,
+    var(--calcite-accordion-item-text-color, var(--calcite-color-text-3))
+  );
 }
 
 :host(:focus),
-:host(:hover) {
+:host(:hover),
+:host(:active),
+:host([expanded]) {
   & .heading {
-    @apply text-color-1;
+    color: var(
+      --calcite-accordion-item-heading-text-color,
```

Comment:
Per our latest guidelines, should these be stateful CSS props?

> component tokens should be added for the appropriate states as well. I.E. if background-color changes on hover then you should also add a `background-color-hover` token. 

Applies to other CSS props affected by state.

Reply:
because all parts of the component respond to the state of the host, we don't need state tokens here.